angular2-jwt.ts

```diff
@@ -1,6 +1,6 @@
 import {Injectable, Injector} from 'angular2/core';
 import {Http, HTTP_PROVIDERS, Headers, BaseRequestOptions, Request, RequestOptions, RequestOptionsArgs, RequestMethod, Response} from 'angular2/http';
-import {Observable} from 'rxjs/Observable';
+import {Observable} from 'rxjs';
```

Comment:
I'm not 100% on the beta 4 rxjs changes, but previous to this, it would mean that you've imported the entire rxjs library on Observable. This is not good, as it ruins anyones chances of minimizing their bundles.